roles/ansible-os-hardening/tasks/minimize_access.yml

```diff
@@ -14,5 +14,5 @@
   file: dest='/etc/shadow' owner=root group=root mode=0600
 
 - name: change su-binary to only be accessible to user and group root
-  file: dest='/bin/su' owner=root group=root mode
+  file: dest='/bin/su' owner=root group=root mode=0750
```

Comment:
I think, ansible should throw an error when a key is provided without a value. I'd rather not change the whole syntax because I like it that way.

I'm going to ask on the ansible mailinglist.